includes/CLI/Plugin_Check_Command.php

```diff
@@ -668,15 +673,22 @@ private function has_runtime_check( array $checks ) {
 	 * @param array $results          Check results.
 	 * @param int   $error_severity   Error severity level.
 	 * @param int   $warning_severity Warning severity level.
+	 * @param bool  $include_low_severity_errors Include less level of severity issues as warning.
+	 * 
+	 * @SuppressWarnings(PHPMD.BooleanArgumentFlag)
 	 * @return array Filtered results.
 	 */
-	private function get_filtered_results_by_severity( $results, $error_severity, $warning_severity ) {
-		$errors = array_filter(
-			$results,
-			function ( $item ) use ( $error_severity ) {
-				return ( 'ERROR' === $item['type'] && $item['severity'] >= $error_severity );
+	private function get_filtered_results_by_severity( $results, $error_severity, $warning_severity, $include_low_severity_errors = false ) {
+		$errors = array();
+		foreach ( $results as $item ) {
+			if ( 'ERROR' === $item['type'] && $item['severity'] >= $error_severity ) {
+				$errors[] = $item;
+			} elseif ( $include_low_severity_errors && $item['severity'] < $error_severity ) {
+				$item['type']     = 'WARNING';
+				$item['severity'] = 10;
+				$errors[]         = $item;
```

Comment:
This doesn't look right, warnings shouldn't be added to the `$errors` list, but the `$warnings` list.

Reply:
ok